### PR TITLE
🔥 Added 38 Twitch emotes

### PR DESCRIPTION
**Pull request by asxlvm**
Hello,
I think I'm speaking for every Twitch fan of the community, we needed some emotes that were missing, I added 38 new emojis visible here;
`PepeClown, Kappa, KEKR, peepoStare, ComfWee, ZULUL, SadgeJuice, SQUADW, peepoSQUAD, peepoWeird, NOOO, Susge, Weirdge, PepeMods, BlackU, peepoChrist, Madge, PagMan, Jebaited, 3Head, 4House, peepoHard, PepePains, 4Heed, AYAYAY, KKool, TriDance, MrCool, MrUncool, SadgeJam, catPOP, PartyKirby, RainbowPls, BoneDance, Dcolon, 1Head, KomodoHype`

These are great additions to the app and I'd be glad if my pull request gets merged, thanks.
Asylum

### CODE REVIEW

**Comment by asxlvm** [comment]
Accidentally closed it, thought it wouldn't add the MrCool and MrUncool there.

**Comment by iamanalt** [comment]
This is a wonderful contribution. Approved!

**Comment by benawad** [comment]
We are capping the # of emotes to what we currently have, you need to remove 38 if you want to add 38.

@theCow61 what does he need to make lowercase?

**Comment by theCow61** [comment]
@benawad the json name and short name need to be lower case.

Also we could use [git lfs](https://git-lfs.github.com/) for the emote assets if the git repo is getting to heavy because of all the images and gifs.

**Comment by BronzW** [comment]
I'm not sure the Twitch emotes (for example `Kappa`) would be allowed in commercial use such as DogeHouse as plenty of sources, including Twitch, state that using Twitch Brand Assets would be against their policy and that their Assets (including **emotes**) are under trademark. _**### Please note that I don't have any legal experience and this is just from what I've read and I might be completely wrong.**_

> Without express permission, we do not allow use of emotes. Twitch and/or third parties own rights such as copyrights, trademarks, and/or rights of publicity in emotes on Twitch. Please note that in addition to us, the use of an emote may require permission from such third parties.

> Don’t incorporate Twitch’s Brand Assets, or variants, in domain names, social media accounts, corporate names, app names, or as a part of any on-line or in person event (including community events or meet-ups), unless we have given express permission

> Don’t combine Twitch Brand Assets as part of a name of a product or service.

> Don’t use Twitch Brand Assets in a way that suggests an affiliation, partnership, sponsorship or endorsement by or with Twitch.
 
> Don’t include Twitch Brand Assets in editorial content that suggests it’s been authored by, or represents the views or opinions of, Twitch or any Twitch personnel or affiliate.

> Don’t present Twitch Brand Assets in a manner that is misleading, defamatory, libelous, obscene, infringing or otherwise in a manner that violates the spirit of Twitch’s Community Guidelines.
 
> Don’t use Twitch Brand Assets on material that infringes the trademark, copyright or any other rights of any third party.

So we are allowed to use BetterTTV emotes. Twitch's own emotes such as Kappa, 4Head, NotLikeThis or LULW would not be allowed on DogeHouse without Twitch's permission so knowing that in the future might be pretty good. 

https://www.twitch.tv/p/sv-se/legal/trademark/#general-usage-guidelines

https://www.twitch.tv/p/sv-se/legal/trademark/#emotes

**Comment by asxlvm** [comment]
Okay so what would I need to do?
Rename shortnames to lowercase
And by the way, Kappa is the emote of a Twitch staff from when Twitch was owned by Justin.tv so I don't think they have Copyright for that,but okay, I'll remove it Sadge
Aaand, why couldn't we host the emote CDN elsewhere so we can have more emotes? Github, etc.

**Comment by asxlvm** [comment]
And 4Head is a FrankerFaceZ emote, so I'll not remove it.

**Comment by asxlvm** [comment]
Aaaand, PogChamp is a Twitch emote kek

**Comment by amitojsingh366** [comment]
yea, we aren't accepting more emotes as of rn...

**Comment by amitojsingh366** [comment]
ok i just read @benawad 's comment so ill reopen this and it seems you didn't understand what he and the others meant. below is an explanation: 
- all the `name` and `short_names` keys in json should be lowercase *(see comment below for example)*
- if you add 38 emotes you will have to remove 38 emotes. basically in the end the total no. of emotes should be less than or equal to 100
- also there's some legality which i have no idea about

File: kofta/src/app/modules/room-chat/EmoteData.ts [hunk 1]
```diff
@@ -1,6 +1,228 @@
 import { EmojiData } from "emoji-mart";
 
 export const customEmojis = [
+  {
+    name: "KomodoHype",
+    short_names: ["KomodoPog"],
```
Comment (amitojsingh366) [inline kofta/src/app/modules/room-chat/EmoteData.ts:6]: @asxlvm these are the keys. so in this case you can just change 
`KomodoHype` to `komodohype` and `KomodoPog` to `komodopog`

Comment (asxlvm) [reply kofta/src/app/modules/room-chat/EmoteData.ts:6]: Okay I see

**Comment by theCow61** [comment]
If you have a lot of emotes you need to lower case, I would recommend making a quick script that takes the json, deserializes it, lowercases the name and short_name, and then serializes it.

Might make a script that does this so people contributing emotes can just run this when after editing the json.

**Comment by jobveldhuis** [comment]
[The Twitch legal trademark notice](https://www.twitch.tv/p/en/legal/trademark/) clearly states that the use of Twitch emotes outside of Twitch is forbidden, unless receiving express permission. I would highly advise against merging this pull request, in any form.

**Comment by asxlvm** [comment]
Okay and? We have PogChamp which is a Twitch emote, and then, how is Discord for ex. not sued as they have no control of use of their CDN, get it?

**Comment by amitojsingh366** [comment]
> Okay and? We have PogChamp which is a Twitch emote, and then, how is Discord for ex. not sued as they have no control of use of their CDN, get it?

but dogehouse does have control over their content. there aren't any custom emotes really. we have to approve all of them and then it goes to the dogehouse repo *(which we have full control over)*. so if what @jobveldhuis said is true then we might get sued. anyway @benawad what do you think abt this?

**Comment by benawad** [comment]
Instead of using twitch emotes, I am going to pay people to make doge themed version of every twitch emote

I don't want any legal problems

**Pull request closed without merging**